controllers/groupController.js

```diff
@@ -1,6 +1,24 @@
 const groupModel = require("../models/groupModel");
 const { isValidString, isEmptyString } = require("../utils/validation");
 
+const list = async (req, res) => {
+  if (!isValidString(req.params.groupId) || isEmptyString(req.params.groupId)) {
+    return res.status(400).send({ message: "[InvalidGroupId] Error occured" });
+  }
+
+  try {
+    const groupResult = await groupModel
+      .findById(req.params.groupId)
+      .populate("keywordIdList", "keyword")
+      .sort({ updatedAt: -1 });
```

Comment:
```suggestion
   const groupId = req.params.groupId;
   
    const groupResult = await groupModel
      .findById(groupId)
      .populate("keywordIdList", "keyword")
      .sort({ updatedAt: -1 });
```
다른 로직들과 동일하게 전달받은 groupId를 변수에 할당시킨 후에 사용하는 게 더 좋을 것 같은데, 어떻게 생각하실까요?!

Reply:
수정했습니다